GitOut/Features/Logging/FileLogger.cs

```diff
@@ -17,12 +17,15 @@ public class FileLogger : ILogger
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
             string content = $"[{DateTime.UtcNow:yyyy-MM-dd HH:mm:ss}] {logLevel}\t{eventId.Id}\t{formatter(state, exception)}\r\n";
-            if (exception != null)
+            if (exception is not null)
             {
                 content += $"{exception}\r\n";
             }
-            Directory.CreateDirectory(Path.GetDirectoryName(filePath));
-            File.AppendAllText(filePath, content);
+            if (Path.GetDirectoryName(filePath) is string directoryName)
+            {
+                Directory.CreateDirectory(directoryName);
+                File.AppendAllText(filePath, content);
+            }
```

Comment:
else throw?
It's a bit tricky though because we can't log it if logging fails 🤪